docs/mesos/Dockerized27Deployment.md

```diff
@@ -0,0 +1,226 @@
+
+Agent (compute) nodes require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| BIRD (BGP)   | 179/tcp           |
+| mesos-agent  | 5051/tcp          |
+
+Example `firewalld` config
+
+    sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
+    sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
+    sudo systemctl restart firewalld
+    
+## 2. Calico
+`calicoctl` is a small CLI tool to control your Calico network.  It's used to start Calico services on your compute host, as well as inspect and modify Calico configuration.
+
+    curl -L -O https://github.com/projectcalico/calico-containers/releases/download/v0.8.0/calicoctl
+    chmod +x calicoctl
+    sudo mv calicoctl /usr/bin/
+
+You can learn more about `calicoctl` by running `calicoctl --help`.
+
+You'll need to configure Calico with the correct location of the etcd service.  In the following line, replace `<MASTER_IP>` with the IP address of the Master node.
+
+    sudo sh -c 'echo ETCD_AUTHORITY=<MASTER_IP>:4001 > /etc/sysconfig/calico'
+
+Then, enable the Calico service via `systemd`
+
+    sudo docker pull calico/node:v0.8.0
+    wget https://raw.githubusercontent.com/projectcalico/calico-mesos/0.27/dockerized-mesos/config/units/calico.service
+    sudo cp calico.service /usr/lib/systemd/system/
+    sudo systemctl enable calico.service
+    sudo systemctl start calico.service
+
+Verify Calico is running
+
+    calicoctl status
+
+## 3. Mesos-Agent
+
+Use the following commands to tell the Mesos Agent where to find Zookeeper.  The Mesos Agent uses Zookeeper to keep track of the current Mesos Master.  We installed it on the same host as the Mesos Master earlier, so substitute the name or IP of that host for `<ZOOKEEPER_IP>`:
+
+    sudo sh -c 'echo ZK=<ZOKEEPER_IP> > /etc/sysconfig/mesos-agent'
+
+You also need to specify the IP address of the Agent to connect to the Mesos cluster.  Run the following command, replacing `<AGENT_IP>` with the Agent's IP address.
+
+    sudo sh -c 'echo IP=<AGENT_IP> >> /etc/sysconfig/mesos-agent'
+
+Then, enable the Mesos Agent service
+
+    sudo docker pull calico/mesos-calico:0.27
+    wget https://raw.githubusercontent.com/projectcalico/calico-mesos/0.27/dockerized-mesos/config/units/mesos-agent.service
+    sudo cp mesos-agent.service /usr/lib/systemd/system/
+    sudo systemctl enable mesos-agent.service
+    sudo systemctl start mesos-agent.service
+
+# Test your cluster
+To ensure that your cluster is properly networking containers with Calico and enforcing policy as expected, run the Calico Mesos Test Framework, which launches various tasks across your Mesos cluster:
+```
+docker run calico/calico-mesos-framework 172.18.8.101:5050
+```
+> NOTE: Some tests require multiple hosts to ensure cross-host communication, and may fail unless you are running 2+ agents.
+
+# Launching Docker Images with Marathon
+Sample `app.json`:
```

Comment:
This section could do with beefing up.  Maybe a little more about Marathon and how to access the UX etc.  obviously we don't want to duplicate Marathon documentation, but since we are suggesting to use Marathon we need more information here.